OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -104,6 +104,18 @@ public SlotDropDownOption(string title, string order, Func<bool> selected)
 			dropdown.ShowDropDown("SPAWN_DROPDOWN_TEMPLATE", 150, spawnPoints, setupItem);
 		}
 
+		/// <summary>Splits a string into two parts on the first instance of a given token.</summary>
+		static Pair<string, string> SplitOnFirstToken(string input, string token = "\\n")
+		{
+			if (string.IsNullOrEmpty(input))
```

Comment:
This would break our tooltip display: we use `null` to represent "no tooltip" while `""` is "tooltip with no text"